src/config/config.php

```diff
@@ -28,8 +28,8 @@
     'paths' => array(
         'api/*' => array(
             'allowedOrigins' => array('*'),
-            'allowedHeaders' => array('*'),
-            'allowedMethods' => array('*'),
+            'allowedHeaders' => array(),
+            'allowedMethods' => array('POST', 'PUT', 'GET', 'DELETE'),
```

Comment:
Ah, I didn't do my homework and check the asm89 code. It doesn't just pass the asterisk along, but handles it to return what was sent in Access-Control-Request-Method. I'll put it back as it was.